### PR TITLE
Introduce new config setting to set a contact email address

### PR DESCRIPTION
**Pull request by sgiehl**
### Description:

There are currently a few places where a user is suggested to contact the administrator of the Matomo instance. In most cases there is a `mailto` link that contains the email addresses of all super users.

In some scenarios it might be unexpected that in this places the (possibly private) mail addresses are disclosed.
To avoid this, this PR introduces as new config setting to set a contact email address, that will be used for those links instead.
By default this config setting isn't set and it falls back to use all super users emails.

In companies for example it might be more helpful to send an email to a support mailbox instead of sending an email to some specific persons.

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

**Comment by Schwuuuuup** [comment]
As the author of the mail to security@matomo.org, that lead to this PR, I just wanted to say thank you. And I hope that the build and Merge Issues can be fixed

File: plugins/CorePluginsAdmin/templates/safemode.twig [hunk 1]
```diff
@@ -128,7 +128,7 @@
         {% else %}
             <p>
                 If this error continues to happen you may want to send an
-                <a href="mailto:{{ emailSuperUser }}?subject={{ 'Fatal error in Matomo ' ~ piwikVersion|e('url') }}&body={{ lastError.message|e('url') }}%20in%20{{ lastError.file|e('url') }}%20{{ lastError.line|e('url') }}%20using%20PHP%20{{ constant('PHP_VERSION') }}">error report</a>
+                <a href="mailto:{{ contactEmail }}?subject={{ 'Fatal error in Matomo ' ~ piwikVersion|e('url') }}&body={{ lastError.message|e('url') }}%20in%20{{ lastError.file|e('url') }}%20{{ lastError.line|e('url') }}%20using%20PHP%20{{ constant('PHP_VERSION') }}">error report</a>
```
Comment (tsteur) [inline plugins/CorePluginsAdmin/templates/safemode.twig]: does it need to be `|e('html_attr')` for contactEmail?

Comment (tsteur) [reply plugins/CorePluginsAdmin/templates/safemode.twig]: below in other ones we use `url` strategy, not sure if that's right

File: plugins/Morpheus/templates/ajaxMacros.twig [hunk 2]
```diff
@@ -13,12 +13,12 @@
...
 
-{% macro requestErrorDiv(emailSuperUser, areAdsForProfessionalServicesEnabled = false, currentModule = '', showMoreHelp = false) %}
+{% macro requestErrorDiv(contactEmail, areAdsForProfessionalServicesEnabled = false, currentModule = '', showMoreHelp = false) %}
     <div id="loadingError">
         <div class="alert alert-danger">
 
-            {% if emailSuperUser is defined and emailSuperUser %}
-                {{ 'General_ErrorRequest'|translate('<a href="mailto:' ~ emailSuperUser ~ '">', '</a>')|raw }}
+            {% if contactEmail is defined and contactEmail %}
+                {{ 'General_ErrorRequest'|translate('<a href="mailto:' ~ contactEmail ~ '">', '</a>')|raw }}
```
Comment (tsteur) [inline plugins/Morpheus/templates/ajaxMacros.twig]: same here should maybe also escape?

**Review by tsteur — CHANGES_REQUESTED** [review]
looks generally good. A UI test will need fixing though https://builds-artifacts.matomo.org/matomo-org/matomo/4.x-dev/50182/UIIntegrationTest_admin_diagnostics_configfile.png

Could you also create an FAQ for this? And where else would we mention this? Maybe in security guide?

File: plugins/CorePluginsAdmin/templates/safemode.twig (hunk 1 above)
Comment (sgiehl) [reply plugins/CorePluginsAdmin/templates/safemode.twig]: tested it a bit and seems `e('url')` works as expected for that case.

**Comment by tsteur** [comment]
@sgiehl if tests pass this should be good to merge

**Pull request merged**